src/main.rs

```diff
@@ -137,7 +137,12 @@ async fn run(do_commit: bool, site_filter: Option<&Regex>) -> anyhow::Result<()>
         .filter(|site| site_filter.map_or(true, |filter| filter.is_match(site.url.as_str())))
         .collect::<Vec<_>>();
     let sites_amount = sites.len();
-    assert!(!sites.is_empty(), "Site filter filtered everything out.");
+    if sites.is_empty() {
+        eprintln!("Error: The Site-filter filtered everything out.
+There are no sites tracked for your query.
+Tipp: Run 'website-stalker run --all' to stalk all tracked sites.");
```

Comment:
The current message is oddly specific to your `run all` case and should also include the option of another filter

```suggestion
Hint: Change the filter or use all sites with 'run --all'.");
```